libcxx/test/std/containers/sequences/vector/vector.modifiers/common.h

```diff
@@ -38,7 +38,35 @@ struct Throws {
 };
 
 bool Throws::sThrows = false;
-#endif
+
+struct ThrowingMove {
+  TEST_CONSTEXPR ThrowingMove() : value(0), do_throw(false) {}
+  TEST_CONSTEXPR explicit ThrowingMove(int v) : value(v), do_throw(false) {}
+  TEST_CONSTEXPR explicit ThrowingMove(int v, bool do_throw) : value(v), do_throw(do_throw) {}
+
+  ThrowingMove(const ThrowingMove& rhs)        = default;
+  ThrowingMove& operator=(const ThrowingMove&) = default;
+
+  TEST_CONSTEXPR_CXX14 ThrowingMove(ThrowingMove&& rhs) : value(rhs.value), do_throw(rhs.do_throw) {
+    if (do_throw)
+      throw 1;
+  }
+  TEST_CONSTEXPR_CXX14 ThrowingMove& operator=(ThrowingMove&& rhs) {
+    value    = rhs.value;
+    do_throw = rhs.do_throw;
+    if (do_throw)
+      throw 1;
+    return *this;
+  }
```

Comment:
```suggestion
  TEST_CONSTEXPR_CXX26 ThrowingMove& operator=(ThrowingMove&& rhs) {
    value    = rhs.value;
    do_throw = rhs.do_throw;
    if (do_throw)
      throw 1;
    return *this;
  }
```

I don't think `throw` is supported by C++14 constexpr. I am not sure clang currently supports `throw` or not. But at least, we need to use `TEST_CONSTEXPR_CXX26`.

Reply:
I think we should keep using `TEST_CONSTEXPR_CXX14`. This `operator=` is not always throwing. When it normally returns (i.e. `do_throw` is `false`), the function call is suitable for constant evaluation in C++14.

Reply:
I think you are right. I've tested on experimental mode with [P3068](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2024/p3068r4.html) enabled ([Godbolt link](https://godbolt.org/z/YhEEfc4Wv)). My test suggests that using `TEST_CONSTEXPR_CXX14` should be fine if constexpr throwing is supported in C++26, and we may fail expectedly without P3068 support.